src/main/java/co/inventorsoft/academy/springresttask/EmailConsole.java

```diff
@@ -0,0 +1,102 @@
+package co.inventorsoft.academy.springresttask;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Almost all of the methods in this class are static. This means that this class is bad candidate to be Spring component.